study.md

```diff
@@ -160,15 +163,23 @@ What is a response? What does it contain? What are some common status codes in a
 response and what do they mean?
 
 ```md
-<!-- your answer here -->
+A response is what the server sends back to the client's request. Responses contain resource representations.
+
+Status codes:
+1xx: request recieved and understood
+2xx: success
+3xx: need addtional action to complete request
+4xx: client error
+5xx: server error
+
 ```
 
 ## Make a curl request
 
 Using curl, how would you get the content from Reddit.com?
 
 ```md
-<!-- your answer here -->
+cURL is a tool that works with URLs
```

Comment:
Yup, but how would you actually make a request to reddit.com? Something like this would work: curl --request GET `http://reddit.com`